### ISSUE TITLE
Incomplete circle?

### ISSUE THREAD

**Issue by CarlosGrohmann** [adamatan/polycircles#1]
Hi, I just found polycircles and it's great!

I'm using to plot a map of earthquakes eipcenters, but the contour of the circles are incomplete, as if the last node of the polygon is not connected with the first. See screenshot:

<img width="645" alt="screen shot 2016-06-30 at 16 30 19" src="https://cloud.githubusercontent.com/assets/1870265/16501631/c0c424d6-3ee0-11e6-9546-d49dfc9b923d.png">

thanks!

**Comment by adamatan** [comment]
I’ll be happy to fix. Could you please give me the data you’re using - which function are you calling and what arguments are you sending it?
Adam

**Comment by CarlosGrohmann** [comment]
Hi, thanks the quick reply. 

Here is a sample of the data:

X,Y,Ano,mm,dd,hh,min,sec,latitude,longitud,prof,erro,MAG,t,Q,Io,Af,st,locality,Reference,field_19,field_20,field_21,field_22,field_23,mag_03
-47.94,-25.01,1789,05,9,,,,-25.010000000000002,-47.939999999999998,0.000000000000000,0,3.700000000000000,4,C,5-6,,SP,CANANEIA,(SANTOS 1850),,,,,,3.400000000000000
-44.88,-21.98,1824,,0,,,,-21.980000000000000,-44.880000000000003,0.000000000000000,0,3.200000000000000,4,C,4-5,,MG,CAXAMBU,,,,,,,2.900000000000000
-45.39,-21.84,1839,07,16,02,,,-21.840000000000000,-45.390000000000001,0.000000000000000,0,3.000000000000000,4,C,4,,MG,CAMPANHA,,,,,,,2.700000000000000
-44.75,-21.03,1840,,0,,,,-21.030000000000001,-44.750000000000000,0.000000000000000,0,3.000000000000000,4,C,4,,MG,BOM SUCESSO,VARIOS EVENTOS,,,,,,2.700000000000000
-43.48,-19.95,1855,07,25,09,,,-19.949999999999999,-43.479999999999997,0.000000000000000,0,3.500000000000000,4,C,5,,MG,S.Joao M.GRANDE,ATUAL B. DE COCAIS,,,,,,3.200000000000000



and this is the code I'm using:


```
import os, sys, csv
from polycircles import polycircles
import simplekml

csvfile = 'sismos_continente2.csv'
datafile = csv.reader(open(csvfile, 'rU'), delimiter=',')
quakelist = list(datafile) # append data to a list

print(quakelist[0])
# ['X', 'Y', 'Ano', 'mm', 'dd', 'hh', 'min', 'sec', 'latitude', 'longitud', 'prof', 'erro', 'MAG', 't', 'Q', 'Io', 'Af', 'st', 'locality', 'Reference', 'field_19', 'field_20', 'field_21', 'field_22', 'field_23', 'mag_03']

# instantiate new KML
kml = simplekml.Kml()

# alpha for color
alpha = 100

# get data from list
for quake in quakelist[1:]:
    # columns from list
    lng = float(quake[0]) # X
    lat = float(quake[1]) # Y
    yyyy = quake[2] # Ano
    magb = quake[12] # MAG
    magw = quake[25] # mag_03
    # create polycircle
    polycircle = polycircles.Polycircle(latitude=lat,
                longitude=lng,
                radius=40000,
                number_of_vertices=36)
    pol = kml.newpolygon(name=yyyy, outerboundaryis=polycircle.to_kml())
    # pol.style.polystyle.outline = 0
    if float(magw) < 3.0:
        # print 'magw < 3.0'
        pol.style.polystyle.color = simplekml.Color.changealphaint(alpha, simplekml.Color.lightyellow)
        pol.style.linestyle.color = simplekml.Color.lightyellow
    elif 3.0 < float(magw) < 3.5:
        # print '3.0 < magw < 3.5'
        pol.style.polystyle.color = simplekml.Color.changealphaint(alpha, simplekml.Color.yellow)
        pol.style.linestyle.color = simplekml.Color.yellow
    elif 3.5 < float(magw) < 4.0:
        # print '3.5 < magw < 4.0'
        pol.style.polystyle.color = simplekml.Color.changealphaint(alpha, simplekml.Color.orangered)
        pol.style.linestyle.color = simplekml.Color.orangered
    else:
        # print 'magw > 4.0'
        pol.style.polystyle.color = simplekml.Color.changealphaint(alpha, simplekml.Color.red)
        pol.style.linestyle.color = simplekml.Color.red

kml.save(wrkDir+'sismos.kml') 
kml = None 

```

**Comment by adamatan** [comment]
Hi,
Thanks again for the report; I've fixed the problem and [added some tests](https://travis-ci.org/adamatan/polycircles). In general, the first point of the polygon is appended to the end of the list of points, so the circle is properly closed.

Could you please download the latest (0.2) polycircles from PyPi and let me know if it works?
Adam

PS Needless to say, [gave you code credit](https://github.com/adamatan/polycircles/blob/master/polycircles/test/test_earthquakes.py#L8):

<img width="572" alt="screen shot 2016-07-01 at 12 38 50 pm" src="https://cloud.githubusercontent.com/assets/1014189/16517954/eb97c962-3f88-11e6-8ce3-ec9dcc552111.png">

**Comment by CarlosGrohmann** [comment]
oops..

```
afbiota:~ guano$ pip install polycircles --upgrade
Collecting polycircles
  Using cached polycircles-0.2.tar.gz
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/private/var/folders/y9/mnj7kbf10pl0zfs74d03sz940000gn/T/pip-build-AEjhSf/polycircles/setup.py", line 6, in <module>
        long_description=open('README.md').read(),
    IOError: [Errno 2] No such file or directory: 'README.md'
```

**Comment by CarlosGrohmann** [comment]
Besides the installation issue (downloading the master branch and installing from the local file works), everything is working fine! Thanks for the quick fix and for adding my code as an example.

**Comment by adamatan** [comment]
Thanks for the feedback. Fixed the issue and upgraded to 0.3. 
Works on my computer, Could you please test on your end?

**Comment by CarlosGrohmann** [comment]
Working fine here!
(closing this ticket)
